src/Integration.Vsix/Commands/HelpMenu/ViewDocumentationCommand.cs

```diff
@@ -0,0 +1,41 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarLint.VisualStudio.IssueVisualization.Helpers;
+
+namespace SonarLint.VisualStudio.Integration.Vsix.Commands
+{
+    internal class ViewDocumentationCommand : VsCommandBase
+    {
+        internal const int ViewDocumentationCommandId = 0x103;
+
+        private readonly IShowInBrowserService showInBrowserService;
+
+        public ViewDocumentationCommand(IShowInBrowserService showInBrowserService)
+        {
+            this.showInBrowserService = showInBrowserService;
```

Comment:
NOTE: I will be making this class more generic, since the code of calling the method can be reused. There's going to be a conflict of Ids if we define them as fields of the class.

Reply:
Not really a conflict, but it will be a bit confusing. We need to rethink the way we define Ids

Reply:
I don't think this is an issue as in the vsct file you can still see all of the IDs for the commands in the command set

Reply:
It would be an issue in the CommandManager class, here:
`this.RegisterCommand(CommonGuids.HelpMenuCommandSet, ViewDocumentationCommand.ViewDocumentationCommandId, new ViewDocumentationCommand(showInBrowserService));`
If I make this class generic, I would need to specify which link to open (rn it only opens the docs link, but you could specify which one with a `Action<IShowInBrowserService>`). Then, having the Id be defined inside the class wouldn't really make sense, because the class is going to be instantiated as different commands with different Ids

Reply:
If you want to use a different link refer to my other PR where I use "IBrowserService" which has  navigate(string url)

Reply:
I am not sure if it is good practice to reuse a command class like this anyways although maybe @rita-sonarsource or @duncanp-sonar have more insight on this :)

Reply:
@georgii-borovinskikh-sonarsource looking at this class, I think adding an abstraction layer for one method might be an overkill and would introduce further complexity. Perhaps let's start with making a command that calls `IBrowserService.Navigate`, I feel that it already does very little. WDYT?

Reply:
I'm not sure I see how using the navigate thing directly would solve the problem of having boilerplate code. How would that look like?

Reply:
It's not really boilerplate code, it's calling one method, and it's already minimal and testable.
Are you proposing a base command that other commands can inherit from, i.e.
```
class GenericCommand
{
	ctor(string url, IBrowserService)
	{
	}

	onInvoke()
	{
		IBrowserService.NavigateToUrl(url);
	}
}

class ViewDocsCommand : GenericCommand
{
	ctor(IBrowserService) : base ("http..", browserService);
}

class ReportAProblemCommand : GenericCommand
{
	ctor(IBrowserService) : base ("http..", browserService);
}
```

In this case I don't think it saves much and it introduces complexity, plus the tests would be exactly the same.
Or are you suggesting something else?

Reply:
Edit: @georgii-borovinskikh-sonarsource I saw that you wrote above "If I make this class generic, I would need to specify which link to open (rn it only opens the docs link, but you could specify which one with a Action<IShowInBrowserService>)." --> In this case I fully disagree as I don't think the command manager should know about URLs.

Reply:
Why do you think it shouldn't know about that?

Reply:
Not about the specific links, but about the configuration of commands

Reply:
You can also do `new GenericCommand(() => showInBrowserService.ShowDocumentation())` which would be the same and "more" generic, but in both cases, I think it's an overkill and doesn't really save much.